pytorch_lightning/lite/lite.py

```diff
@@ -399,16 +399,19 @@ def seed_everything(seed: Optional[int] = None, workers: Optional[bool] = None)
         return seed_everything(seed=seed, workers=workers)
 
     def _run_impl(self, run_method: Callable, *args: Any, **kwargs: Any) -> Any:
-        self._strategy.setup_environment()
-
         # apply sharded context to prevent OOM
         run_method = partial(self._run_with_sharded_context, run_method)
+        run_method = partial(self._wrapped_run_method, run_method)
```

Comment:
i don't think we need this, it can be merged with run_with_sharded_context if necessary